test/modules/packages/Crypto/saru/blowfish/bfTest2.skipif

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Could you make this a symlink to aesTest1.skipif like we did in https://github.com/chapel-lang/chapel/pull/7377?